src/assets/scripts/main.js

```diff
@@ -1,120 +1,165 @@
 /* Functions
 ============================================================================ */
 
-function formValidator() { 
-
-  let checkForm = function (form) {
-    
-    console.log(form);
+function FormValidator() {
+  const checkForm = (form) => {
+    const $errorMessage = form.querySelector('.image-form__error-message');
```

Comment:
@finnge warum hier `$errorMessage` statt `errorMessage`?

Reply:
Ich habe es mir als Konvention angewöhnt DOM-Elemente mit einem `$` zu prefixen. ✌🏼 
So sind sie von _normalen_ Variablen schön zu unterscheiden.

Reply:
Auch wenn ich gerade sehe, dass ich das bei `formInput` in Zeile 10 nicht gemacht haben 🤪